mllib/src/test/scala/org/apache/spark/ml/clustering/GaussianMixtureSuite.scala

```diff
@@ -130,4 +230,29 @@ object GaussianMixtureSuite {
     "maxIter" -> 2,
     "tol" -> 0.01
   )
+
+  val denseData = Seq(
```

Comment:
We should copy test cases and data from mllib to ml when we copying algorithm implementation. Since the spark.mllib package would be deleted eventually(may be in 3.0?), we should keep spark.ml includes all necessary data.